src/main/msp/msp_protocol.h

```diff
@@ -285,8 +285,20 @@
 #define MSP_VTXTABLE_POWERLEVEL  138    //out message         vtxTable powerLevel data
 #define MSP_MOTOR_TELEMETRY      139    //out message         Per-motor telemetry data (RPM, packet stats, ESC temp, etc.)
 
-#define MSP_SIMPLIFIED_TUNING       140    //out message         Simplified tuning values and enabled state
-#define MSP_SET_SIMPLIFIED_TUNING   141    //in message          Set simplified tuning positions and apply the calculated tuning
+#define MSP_SIMPLIFIED_TUNING             140    //out message         Simplified tuning values and enabled state
+#define MSP_SET_SIMPLIFIED_TUNING         141    //in message          Set simplified tuning positions and apply the calculated tuning
+
+#define MSP_CALCULATE_SIMPLIFIED_PIDS_TUNING           142    //out message      Requests calculations of PID values based on sliders. Sends the calculated values back. But don't save anything to the firmware
+#define MSP_CALCULATE_SIMPLIFIED_GYRO_FILTERS_TUNING   143    //out message      Requests calculations of gyro filter values based on sliders. Sends the calculated values back. But don't save anything to the firmware
+#define MSP_CALCULATE_SIMPLIFIED_DTERM_FILTERS_TUNING  144    //out message      Requests calculations of gyro filter values based on sliders. Sends the calculated values back. But don't save anything to the firmware
+
+#define MSP_ARE_SIMPLIFIED_PIDS_VALID            145 // out message...... Returns TRUE to the requester if PIDF values match with sliders. FALSE otherwise
+#define MSP_ARE_SIMPLIFIED_GYRO_FILTERS_VALID    146 // out message...... Returns TRUE to the requester if Gyro Filter values match with sliders. FALSE otherwise
+#define MSP_ARE_SIMPLIFIED_DTERM_FILTERS_VALID   147 // out message...... Returns TRUE to the requester if Dterm Filter values match with sliders. FALSE otherwise
+
+#define MSP_SET_SIMPLIFIED_PIDS_TUNING           148    //out message         Set simplified PIDs tuning positions and apply the calculated tuning
+#define MSP_SET_SIMPLIFIED_GYRO_FILTERS_TUNING   149    //out message         Set simplified Gyro Filters tuning positions and apply the calculated tuning
+#define MSP_SET_SIMPLIFIED_DTERM_FILTERS_TUNING  150    //out message         Set simplified Dterm Filters tuning positions and apply the calculated tuning
```

Comment:
```suggestion
```
MSPCode 150 is already used by
#define MSP_STATUS_EX 150 ?

Reply:
fixed